taxi_service/settings.py

```diff
@@ -143,3 +144,7 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap4"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The `CRISPY_TEMPLATE_PACK` setting is defined twice (lines 80 and 150). Consider removing one of these duplicate lines to avoid redundancy and potential confusion.